### PR TITLE
Notes and links on fare calculator

### PR DESCRIPTION
**Pull request by alicia-mbta**
#### Summary of changes
**Asana Tickets:**
[Fare Calculator Accordion | Evergreen notations](https://app.asana.com/0/555089885850811/1169236529454071)
[Fare Calculator Accordion | Transfers notes](https://app.asana.com/0/555089885850811/1169236529454073)

**THIS NEEDS TO BE MERGED AFTER #504**

### CODE REVIEW

File: apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex [hunk 1]
```diff
@@ -58,3 +58,15 @@
    </span>
   </div>
 </div>
+
+<div class="m-trip-plan-farecalc__transfer-block">
+  <span class="m-trip-plan-farecalc__transfer-title">Reduced fares:</span> Some riders are eligible for <a href="https://www.mbta.com/reduced-fares" target="_blank">reduced fares</a>. Military personnel and children 11 and under ride for free.
```
[the remark this answers is not in the document]
Comment (thecristen) [reply apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex:63]: https://www.mbta.com/reduced-fares appears to lead to a 404

File: apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex [hunk 2]
```diff
@@ -58,3 +58,15 @@
...
+</div>
+
+<div class="m-trip-plan-farecalc__transfer-block">
+  <span class="m-trip-plan-farecalc__transfer-title">Transfers:</span> This trip suggestion includes one or more transfers. <a href="https://www.mbta.com/transfers" target="_blank">Learn more about transfers</a>
+</div>
```
Comment (thecristen) [inline apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex:68]: This shouldn't show for itineraries that have no transfers

**Comment by arkadyan** [comment]
#504 is now merged. You might need to rebase against `feature/fare-calculator-accordion`.

File: apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex [hunk 3]
```diff
@@ -79,3 +79,16 @@
...
+
+<%= if transfer_note != nil do %>
+  <div class="m-trip-plan-farecalc__notes-block">
+    <span class="m-trip-plan-farecalc__title">Transfers:</span> This trip suggestion includes one or more transfers. <%= link "Learn more about transfers", to: fare_path(@conn, :show, "transfers") %>.
+  </div>
+<% end %>
+
+<div>
+  <%= link "See fares overview", to: fare_path(@conn, :show, "") %>
```
Comment (thecristen) [inline apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex:93]: Sorry to have missed this before, but let's add the `.c-call-to-action` class to this link to get the nice underline!

Comment (alicia-mbta) [reply apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex:93]: Sure! I thought the underline in Zeroheight was just to indicate it was a hyperlink :sweat_smile: Changing this.

**Pull request merged**